katana/native/file_manager/file_manager_utils/ftp_utils.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python3
+import os
+import ftplib
+from ftplib import FTP
+from katana.native.file_manager.views import getpath
+
+def ftpfile(host, port, username, passwd, destdir, files_path, files_name, all_files_path, request):
```

Comment:
The argument ```request``` is not being utilized. Please remove it.

Reply:
It is. Check Line No 23.